ibis/backends/snowflake/tests/test_client.py

```diff
@@ -372,25 +372,27 @@ def test_timestamp_memtable(con):
     tm.assert_frame_equal(result, df)
 
 
-def test_connect_without_snowflake_url(con):
+def test_connect_without_snowflake_url():
     # We're testing here that the non-URL connection works.
     # Specifically that a `database` location passed in as "catalog/database"
     # will be parsed correctly
-    user = con.con.user
-    account = con.con.account
-    catalog = con.con.database
-    database = con.con.schema
-    warehouse = con.con.warehouse
+    user = os.getenv("SNOWFLAKE_USER")
+    account = os.getenv("SNOWFLAKE_ACCOUNT")
+    catalog = os.getenv("SNOWFLAKE_DATABASE")
+    database = os.getenv("SNOWFLAKE_SCHEMA")
+    warehouse = os.getenv("SNOWFLAKE_WAREHOUSE")
 
     if (password := os.getenv("SNOWFLAKE_PASSWORD")) is None:
         pytest.mark.skip(reason="No snowflake password set, nothing to do here")
```

Comment:
When inside a test, you'll want to do `pytest.skip` instead of `pytest.mark.skip`. The former raises and skips the test, the latter won't do anything (and later on you'll get an error).

Reply:
ahh, nice catch, thanks.